packages/pods-core/src/builtin/GraphvizPod.ts

```diff
@@ -0,0 +1,161 @@
+import fs from "fs-extra";
+import _ from "lodash";
+import { DLink, NoteProps, NoteUtils } from "@dendronhq/common-all";
+import path from "path";
+import { ExportPod, ExportPodPlantOpts } from "../basev3";
+import { ExportPodConfig } from "../basev3";
+
+const ID = "dendron.graphviz";
+
+interface ParentDictionary {
```

Comment:
prefer using `type` to `interface`

- types and interfaces are mostly interchangable, `types` are a bit more flexible and so we've standardized on using it
- see official typescript docs [here](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)